code/modules/client/client_procs.dm

```diff
@@ -219,11 +225,15 @@
 	//* Setup user interface
 	// todo: move top level menu here, for now it has to be under prefs.
 	// Instantiate statpanel
-	addtimer(CALLBACK(src, PROC_REF(statpanel_boot)), 0)
+	spawn(1)
+		statpanel_boot()
 	// Instantiate tgui panel
 	tgui_panel = new(src, "browseroutput")
 	// Instantiate cutscene system
-	addtimer(CALLBACK(src, PROC_REF(init_cutscene_system)), 0)
+	spawn(1)
+		init_cutscene_system()
+	// instantiate tooltips
+	tooltips = new(src)
```

Comment:
tl;dr:

- timers don't fire fast enough because we don't have MC init stages yet
- spawn(0) is too fast
- so spawn(1) is kind of the best for these procs
- realistically all of these should be datums but i'll leavei t for the TGUI refresh PR.